### PR TITLE
Sync SQL with IM v3.1

### PR DESCRIPTION
**Pull request by douradofrazer**
Resolves:
https://dcsa.atlassian.net/browse/DDT-40

### CODE REVIEW

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 1]
```diff
@@ -316,7 +323,6 @@ CREATE TABLE dcsa_im_v3_0.equipment (
     equipment_reference varchar(15) PRIMARY KEY,    -- The unique identifier for the equipment, which should follow the BIC ISO Container Identification Number where possible. According to ISO 6346, a container identification code consists of a 4-letter prefix and a 7-digit number (composed of a 3-letter owner code, a category identifier, a serial number and a check-digit). If a container does not comply with ISO 6346, it is suggested to follow Recommendation #2 “Container with non-ISO identification” from SMDG.
     -- Unique code for the different equipment size/type used for transporting commodities. The code is a concatenation of ISO Equipment Size Code and ISO Equipment Type Code A and follows the ISO 6346 standard.
     iso_equipment_code char(4) NULL REFERENCES dcsa_im_v3_0.iso_equipment_code (iso_equipment_code),
-    is_shipper_owned boolean NOT NULL DEFAULT false,
```
Comment (nt-as) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:319]: This is used by EBL. If it is removed, then we need a replacement for EBL.

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 2]
```diff
@@ -508,20 +514,19 @@ CREATE TABLE dcsa_im_v3_0.document_type (
...
-    empty_indicator_code varchar(5) PRIMARY KEY,
-    empty_indicator_name varchar(100) NOT NULL
+    empty_indicator_code varchar(5) PRIMARY KEY
 );
 
 DROP TABLE IF EXISTS dcsa_im_v3_0.event CASCADE;
 CREATE TABLE dcsa_im_v3_0.event (
     event_id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
-    event_type text NOT NULL,
+--  event_type text NOT NULL,
```
Comment (nt-as) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:529]:
```suggestion
```

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql (hunk 1 above)
Comment (douradofrazer) [reply datamodel/initdb.d/03_dcsa_im_v3_0.sql:319]: @raisoman  had commented that it's not supposed to be in equipment, maybe he can clarify.
However note that `is_shipper_owned` its not part of `equipment` in IM 3.1.

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql (hunk 2 above)
Comment (douradofrazer) [reply datamodel/initdb.d/03_dcsa_im_v3_0.sql:529]: Do we need this? as we are aware at the time of fetching the data which type of event it is.

Comment (nt-as) [reply datamodel/initdb.d/03_dcsa_im_v3_0.sql:529]: I meant drop the line entirely rather than leaving it commented out.

**Review by nt-as — APPROVED** [review]

**Pull request merged**